demos/python_demos/action_recognition/models.lst

```diff
@@ -3,3 +3,4 @@ action-recognition-????-decoder
 action-recognition-????-encoder
 driver-action-recognition-adas-????-decoder
 driver-action-recognition-adas-????-encoder
+weld-porosity-detection-????
```

Comment:
You'll need to remove this for now, as this'll make the demo tests fail until we publish the actual model files. I'll try to readd it when I do that.